examples/RL/ppo.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+import os
+
+from stable_baselines3 import PPO
+from stable_baselines3.common.callbacks import CheckpointCallback, EvalCallback
+from stable_baselines3.common.env_util import make_vec_env
+from stable_baselines3.common.utils import get_linear_fn
+from stable_baselines3.common.vec_env import SubprocVecEnv
+
+from l5kit.environment.feature_extractor import CustomFeatureExtractor
+
+
+os.environ["L5KIT_DATA_FOLDER"] = "/tmp/l5kit_data"
```

Comment:
I think it would be better to check if this environment variable is present and then throw a warning, instead of assuming that the user will always put data into `/tmp/l5kit_data`. Also, if we want to call this example from another place and set this environment variable before it, it will be overwritten after it starts executing. I would leave for the user/caller of this script the responsibility of setting the `L5KIT_DATA_FOLDER`. What happens today if this is not set ? Is there any warning already ? If yes, then we can just remove it, if there is an error that is not clear, then we should check and throw a clear error message.

Reply:
I have added the following check:
```
if "L5KIT_DATA_FOLDER" not in os.environ:
    raise KeyError("L5KIT_DATA_FOLDER environment variable not set")

```